### ISSUE TITLE
Add Map.TakeSnapshot method

### ISSUE THREAD

**Issue by amay077** [amay077/Xamarin.Forms.GoogleMaps#241]
### SUMMARY

Add ``Map.TakeSnapshot`` method based on [Google Maps Android API](http://bit.ly/2lZM1h1) .

### DETAILS

```
// Method define in Map class
Task<Image> TakeSnapshot()


// Usage
Image snapshot = await map.TakeSnapshot();
```

I should consider 2nd arg in ``public final void snapshot (GoogleMap.SnapshotReadyCallback callback, Bitmap bitmap)``.

### PLATFORMS

- [x] Android
- [x] iOS
- [ ] UWP

**Comment by DavidPressman** [comment]
Very interested in this enhancement. Any thoughts on when it might become available?

**Comment by amay077** [comment]
I released [v1.8.0](https://www.nuget.org/packages/Xamarin.Forms.GoogleMaps/1.8.0) that contained it.
